src/sql/workbench/contrib/query/browser/keyboardQueryActions.ts

```diff
@@ -157,15 +157,16 @@ export class CopyQueryWithResultsKeyboardAction extends Action {
 				let resultSummary = queryRunner.batchSets[0].resultSetSummaries[i];
 				let result = await queryRunner.getQueryRows(0, resultSummary.rowCount, resultSummary.batchId, resultSummary.id);
 				let tableHeaders = resultSummary.columnInfo.map((col, i) => (col.columnName));
-				let htmlTableHeaders = resultSummary.columnInfo.map((col, i) => (`<th style="border:solid black 1.0pt; whiteSpace:nowrap">${escape(col.columnName)}</th>`));
+				let htmlTableHeaders = resultSummary.columnInfo.map((col, i) => (`<td style="border:1.0px solid black;padding:3pt;font-size:9pt;font-weight: bold;">${escape(col.columnName)}</td>`));
```

Comment:
Why are you changing this to td? th should be used for headers so not sure what this change is fixing

Reply:
And the header should be a row itself, so it should look like this generally : 

```
<tr>
   <th>Co1</th>
   <th>Col2</th>
</tr>
```

https://www.w3schools.com/TAGs/tag_th.asp

Reply:
Oh nevermind you add it below. I guess I'd suggest keeping the tr bit here since that's what you do the rest instead of appending it later on. But it's fine if you'd prefer to keep it like this.

Reply:
This was done to resolve the notebook copy paste issue. I have added the context in the description as well.

Reply:
mmmm, that seems like a Notebook issue to me then. When you paste it there is it pasting the HTML content which is then rendered by the Notebook cell?

@chlafreniere

Reply:
So, it doesn't paste the HTML content when th tags are used for column headers. When I change it to td tags which I have in this PR, then it properly copies the HTML content. Like this:
![image](https://user-images.githubusercontent.com/11342435/101202027-85b33d80-361d-11eb-863f-2658d08db144.png)

Reply:
Can you paste that HTML text in here? That definitely sounds like a notebook bug to me if that's the case.

Reply:
Hm, basic table seems to work fine for me : 

![image](https://user-images.githubusercontent.com/28519865/101202497-34577e00-361e-11eb-9fd0-c97c27cbba76.png)

Maybe something with the extra formatting you're adding?

Reply:
@gupta1, as @Charles-Gagnon was saying, with sample HTML we can diagnose the issue very quickly on the notebooks side.

Reply:
hmm so I have tried this before. Pasting the html directly in markdown works fine in ADS notebooks. It happens when we paste it to Rich text view. So here is the html generated when I use th tags : 

"[@&quot;1ES Day Attendees&quot;]<br /> | take 2 | project BoothNumber, AddressBookTitleDesc<div><br/><br/>
				<table cellPadding="5" cellSpacing="1" style="border:1;border-color:Black;font-family:Segoe UI;font-size:12px;border-collapse:collapse">
				<tr style="background-color:DarkGray"><th style="border:1.0px solid black;padding:3pt;font-size:9pt;font-weight: bold;">BoothNumber</th><th style="border:1.0px solid black;padding:3pt;font-size:9pt;font-weight: bold;">AddressBookTitleDesc</th></tr><tr><td style="border:1.0px solid black;padding:3pt;font-size:9pt;">1</td><td style="border:1.0px solid black;padding:3pt;font-size:9pt;">SOFTWARE ENGINEER II</td></tr><tr><td style="border:1.0px solid black;padding:3pt;font-size:9pt;">1</td><td style="border:1.0px solid black;padding:3pt;font-size:9pt;">PROGRAM MANAGER 2</td></tr>
				</table></div>"

Currently we can repo this bug by using this command to copy paste results (Ctrl + Alt + C) and pasting it in rich text view of notebook cell. If we change it to split view, there is no HTML content

Reply:
If you share it as code in markdown it should work I believe.

Reply:
Just put it in a code block like I did for my example above 

```
<tr>
   <th>Co1</th>
   <th>Col2</th>
</tr>
```

Yeah so pasting directly into rich tech view doesn't work and I'm not sure it should. But if you paste into the MD section then it renders correctly for me.

Reply:
yeah, RTF files does support table but not in html. this won't for rtf is by design.

Reply:
`'[@&quot;1ES Day Attendees&quot;]<br /> | take 2 | project BoothNumber, AddressBookTitleDesc<div><br/><br/>
				<table cellPadding="5" cellSpacing="1" style="border:1;border-color:Black;font-family:Segoe UI;font-size:12px;border-collapse:collapse">
				<tr style="background-color:DarkGray"><th style="border:1.0px solid black;padding:3pt;font-size:9pt;font-weight: bold;">BoothNumber</th><th style="border:1.0px solid black;padding:3pt;font-size:9pt;font-weight: bold;">AddressBookTitleDesc</th></tr><tr><td style="border:1.0px solid black;padding:3pt;font-size:9pt;">1</td><td style="border:1.0px solid black;padding:3pt;font-size:9pt;">SOFTWARE ENGINEER II</td></tr><tr><td style="border:1.0px solid black;padding:3pt;font-size:9pt;">1</td><td style="border:1.0px solid black;padding:3pt;font-size:9pt;">PROGRAM MANAGER 2</td></tr>
				</table></div>'`

Reply:
So given the findings in https://github.com/microsoft/azuredatastudio/issues/13681 it seems like a fix you could (and should) do here is add the thead wrapping the th elements.

e.g. 
```
<table>
  <thead>
    <tr>
      <th>Month</th>
      <th>Savings</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>January</td>
      <td>$100</td>
    </tr>
    <tr>
      <td>February</td>
      <td>$80</td>
    </tr>
  </tbody>
</table>
```